### PR TITLE
Social hub page

### PR DESCRIPTION
**Pull request by hflynn**
Takes over from #118 

Staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/on-the-web/

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -35,12 +35,13 @@
                     </ul>
                 </div>
                 <div class="medium-2 columns end">
-                    <h6>We’re on social media</h6>
+                    <h6>OME Public</h6>
                     <ul class="menu vertical">
                         <li><a href="https://twitter.com/openmicroscopy" target="_blank"><i class="fa fa-fw fa-twitter"></i> Twitter</a></li>
                         <li><a href="https://www.linkedin.com/company/ome---open-microscopy-environment" target="_blank"><i class="fa fa-fw fa-linkedin"></i> LinkedIn</a></li>
                         <li><a href="https://www.facebook.com/openmicroscopy" target="_blank"><i class="fa fa-fw fa-facebook"></i> Facebook</a></li>
                         <li><a href="http://webchat.freenode.net/?channels=%23ome&uio=d4" target="_blank"><i class="fa fa-fw fa-comments"></i> IRC</a></li>
```
Comment (mtbc) [inline _includes/footer.html]: Outwith this PR, interesting that GitHub highlights a character here. Are we meant to encode using `&amp;`?

File: on-the-web/index.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            </div>
+        </section>
+        <hr class="whitespace"> 
+        <div class="marketing-site-content-section">                       
+            <div class="marketing-site-content-section-img">
+                <img src="{{ site.baseurl }}/img/stock/pixabay/laptop-notes.jpg" alt="team meeting notes" />
+            </div>
+            <div class="marketing-site-content-section-block">
+                <h3 class="marketing-site-content-section-block-header">OME Team Meeting Minutes</h3>
+                <p class="marketing-site-content-section-block-subheader subheader">We meet every Tuesday at 2pm GMT to discuss the current progress of the project.</p>
```
Comment (mtbc) [inline on-the-web/index.html]: It's GMT for only half the year.

File: _includes/footer.html (hunk 1 above)
Comment (hflynn) [reply _includes/footer.html]: It is indeed not the only example and AFAIK all the relevant links work fine despite the highlighting

Comment (mtbc) [reply _includes/footer.html]: https://stackoverflow.com/questions/3705591/do-i-encode-ampersands-in-a-href suggests it *should* be encoded; perhaps our browsers are simply kindly working around the brokenness. I can create a card.

File: on-the-web/index.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        <hr class="invisible">
+        <section class="social-media-feature-section">
+            <div class="social-media-feature-section-outer">
+                <h3 class="social-media-feature-section-headline"><i class="fa fa-twitter" aria-hidden="true"></i> Follow Us on Twitter</h3>                
+                <div class="social-media-feature-section-inner">                
+                    <div class="social-media-feature-section-feature top-left">
+                        <a class="account-profile" href="http://www.twitter.com/openmicroscopy" target="_blank"><img class="account-profile-pic" src="https://pbs.twimg.com/profile_images/430266361785229313/MU4Sx0sn_400x400.png" alt="@openmicroscopy twitter profile pic"></a>
+                        <div>
+                            <p class="account-handle"><a href="http://www.twitter.com/openmicroscopy" target="_blank">@openmicroscopy</a></p>
+                            <p class="account-desc">Our main twitter account for project-wide news and interest stories</p>
```
Comment (mtbc) [inline on-the-web/index.html]: Initial capital for "Twitter"?

**Comment by mtbc** [comment]
Thank you for clickable icons. 👍

File: on-the-web/index.html [hunk 4]
```diff
@@ -0,0 +1,89 @@
+---
+layout: default-no-anchor
+title: On the Web
+---          
+        <div class="callout large primary" id="bg-image-social" style="margin-bottom: 0rem;">
```
Comment (mtbc) [inline on-the-web/index.html]: Should I be seeing this background image in staging?

File: on-the-web/index.html [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+                        <div>
+                            <p class="account-handle"><a href="https://github.com/openmicroscopy" target="_blank">@openmicroscopy</a></p>
+                            <p class="account-desc">The main GitHub organization for OMERO, Bio-Formats, documentation and infrastructure management.</p>
+                        </div>
+                    </div>  
+                    <div class="social-media-feature-section-feature">
+                        <a class="account-profile" href="https://github.com/ome" target="_blank"><img class="account-profile-pic" src="https://avatars2.githubusercontent.com/u/975861?v=4&s=200" alt="@OME github profile pic"></a>
+                        <div>
+                            <p class="account-handle"><a href="https://github.com/ome" target="_blank">@OME</a></p>
+                            <p class="account-desc">The home of OME Files, OMERO.web apps, scripts, and various Java libraries. </p>
```
Comment (mtbc) [inline on-the-web/index.html]: These end with periods but the ones near the top of the page don't.

**Comment by mtbc** [comment]
Apart from minor nitpicks / queries, looking generally good!

File: on-the-web/index.html (hunk 4 above)
Comment (hflynn) [reply on-the-web/index.html]: it's the banner image behind the 'On the Web' heading

Comment (mtbc) [reply on-the-web/index.html]: on-the-web

Comment (mtbc) [reply on-the-web/index.html]: Huh, it *does* appear in Chromium. (The background images on other pages appear fine for me in Firefox.)

Comment (hflynn) [reply on-the-web/index.html]: Ah, I'd only looked at it on Safari, any ideas @lunson ?

Comment (hflynn) [reply on-the-web/index.html]: Hmm, I'm seeing a slight delay in loading for Firefox but it does work on 55.0.3

Comment (mtbc) [reply on-the-web/index.html]: Huh, yes, works fine for me in 55.0.3 from my laptop at home, failing only in the 52.3.0 here at work. Maybe just a weirdness that won't be a general problem.

File: _includes/footer.html (hunk 1 above)
Comment (mtbc) [reply _includes/footer.html]: Maybe it's something odd with our local networking: whenever I look at the diff on GH then for this image I get,
social

File: on-the-web/index.html (hunk 4 above)
Comment (mtbc) [reply on-the-web/index.html]: Maybe it's something odd with networking: whenever I look at the diff on GH then for this image I get,
social

File: _includes/footer.html (hunk 1 above)
Comment (hflynn) [reply _includes/footer.html]: Loads fine for me!

**Comment by hflynn** [comment]
Merging this so branches can be updated to fix conflicts

**Pull request merged**